src/BloomBrowserUI/bookEdit/TopicChooser/TopicChooser.js

```diff
@@ -39,8 +37,8 @@ var TopicChooser = (function () {
                             var translationGroup = $("div[data-book='topic']").parent();
                             var englishDiv = translationGroup.find("[lang='en']")[0];
                             if (!englishDiv) {
-                                englishDiv = translationGroup.find("div[data-book='topic']").first().clone();
-                                $(englishDiv).attr("lang", "en");
+                                englishDiv = translationGroup.find("div[data-book='topic']")[0];
+                                $(englishDiv).clone().attr("lang", "en");
```

Comment:
Yes, the "englishDiv = ..." line was setting a jQuery object to a variable previously defined as HTMLElement.